### ISSUE TITLE
Database Periodically Crashes

### ISSUE THREAD

**Issue by mangareader645** [Nagru/Manga-Organizer#4]
I have well over 400 mangas that I'm using this to keep track of. I've had this issue a few times now and it usually does not fix itself. What happens is basically one night I can be using it and saving new mangas and it works just fine. I save and close it for the night, and in the morning when I run it the window and process will pop up but you can not view the window at all. It basically is as if it has just crashed but it is still running in the background.

**Comment by Nagru** [comment]
Can you confirm that you're using the latest version? (Check the about tab and let me know what the number is). Is there any pattern to the issue; is "in the morning" typically when it happens? What OS are you running? How long has this been happening for? And would you send me a copy of your MangaDB.sqlite file?

This isn't an issue I've personally seen, so I'm going to need as much info as you can give to figure out the scenario.

**Comment by mangareader645** [comment]
I'm running version 4.6.71 and on Windows 7. A copy of my broken database is available on Google Drive https://drive.google.com/file/d/0B6dnrUo26g7_c000MW9vN3BNVVU/view. It doesn't happen all that often, but once it happens the application is unable to load it from that point on requiring me to re-enter all the data.

**Comment by Nagru** [comment]
Thank you! Turns out it's an issue with remembering the last location of the program on your screen when starting up again. Somehow in your case it ended up with X and Y coordinates of -3200px. I'll go through the code and patch this up.

In the interim, I've updated Settings.FormPosition in your DB so it will show up on your screen when you load. The data is still safe: <a href="https://cloud.openmailbox.org/public.php?service=files&t=43967164da061f6e88a40500174a58ee&download">(this link expires in a couple of days)</a>
